src/map/packets_keys.h

```diff
@@ -26,496 +26,2110 @@
+
+// 2017-01-04bRagexeRE
+#if PACKETVER == 20170104
+	packetKeys(0x44416BC3,0x6C8D1817,0x072D75D5);
+#endif
+
+// 2017-01-11aRagexeRE
+#if PACKETVER == 20170111
+	packetKeys(0x19B637F9,0x0E9C378A,0x41673186);
 #endif
 
+// 2017-01-18aRagexe, 2017-01-18aRagexeRE
+#if PACKETVER == 20170118
+	packetKeys(0x456B36EB,0x15EB34B0,0x18C42E43);
+#endif
+
+// 2017-01-25aRagexe, 2017-01-25aRagexeRE
 #if PACKETVER == 20170125
-	packetKeys(0x066E04FE,0x3004224A,0x04FF0458); /* 4144 */
+	packetKeys(0x066E04FE,0x3004224A,0x04FF0458);
+#endif
+
+// 2017-02-01aRagexe, 2017-02-01aRagexeRE
+#if PACKETVER == 20170201
+	packetKeys(0x2011228E,0x00453005,0x628E7F0A);
 #endif
 
+// 2017-02-08aRagexeRE, 2017-02-08bRagexeRE
 #if PACKETVER == 20170208
-	packetKeys(0x6A764E5F,0x0609570D,0x28AE07FA); /* 4144 */
+	packetKeys(0x6A764E5F,0x0609570D,0x28AE07FA);
+#endif
+
+// 2017-02-15aRagexeRE
+#if PACKETVER == 20170215
+	packetKeys(0x50C01315,0x51012A42,0x679E16C3);
+#endif
+
+// 2017-02-22aRagexe, 2017-02-22aRagexeRE
+#if PACKETVER == 20170222
+	packetKeys(0x5A242F2F,0x6C556187,0x73C022CB);
 #endif
 
+// 2017-02-28aRagexe, 2017-02-28aRagexeRE, 2017-02-28bRagexe
 #if PACKETVER == 20170228
-	packetKeys(0x771D4F2B,0x20EF1F4C,0x0D5135C8); /* 4144 */
+	packetKeys(0x771D4F2B,0x20EF1F4C,0x0D5135C8);
 #endif
 
+// 2017-03-08bRagexeRE
 #if PACKETVER == 20170308
-	packetKeys(0x653470A9,0x6B316A71,0x5C712C71); /* 4144 */
+	packetKeys(0x653470A9,0x6B316A71,0x5C712C71);
 #endif
 
+// 2017-03-15cRagexe, 2017-03-15cRagexeRE
 #if PACKETVER == 20170315
-	packetKeys(0x399A0856,0x56642A94,0x7F77157D); /* 4144 */
+	packetKeys(0x399A0856,0x56642A94,0x7F77157D);
 #endif
 
+// 2017-03-22aRagexeRE, 2017-03-22bRagexeRE
 #if PACKETVER == 20170322
-	packetKeys(0x2050167B,0x01731233,0x40337033); /* 4144 */
+	packetKeys(0x2050167B,0x01731233,0x40337033);
 #endif
 
+// 2017-03-29cRagexe, 2017-03-29cRagexeRE, 2017-03-29dRagexeRE
 #if PACKETVER == 20170329
-	packetKeys(0x18B31A80,0x1B0B1D56,0x16857D6A); /* 4144 */
+	packetKeys(0x18B31A80,0x1B0B1D56,0x16857D6A);
 #endif
 
+// 2017-04-05bRagexe, 2017-04-05bRagexeRE, 2017-04-05cRagexe, 2017-04-05cRagexeRE
 #if PACKETVER == 20170405
-	packetKeys(0x1FDE7DAC,0x2F9F5B63,0x3F2062AF); /* 4144 */
+	packetKeys(0x1FDE7DAC,0x2F9F5B63,0x3F2062AF);
+#endif
+
+// 2017-04-12aRagexe, 2017-04-12aRagexeRE
+#if PACKETVER == 20170412
+	packetKeys(0x39223393,0x5C847779,0x10217985);
+#endif
+
+// 2017-04-19bRagexeRE
+#if PACKETVER == 20170419
+	packetKeys(0x1F8F4B3F,0x2E481F03,0x39ED4178);
+#endif
+
+// 2017-04-26dRagexeRE
+#if PACKETVER == 20170426
+	packetKeys(0x167642A7,0x1DEC3D26,0x6D046D4C);
+#endif
+
+// 2017-05-02dRagexeRE
+#if PACKETVER == 20170502
+	packetKeys(0x05224194,0x466D4204,0x31F02EE0);
 #endif
 
+// 2017-05-17aRagexeRE, 2017-05-17bRagexeRE
 #if PACKETVER == 20170517
-	packetKeys(0x2CC4749A,0x1FA954DC,0x72276857); /* 4144 */
+	packetKeys(0x2CC4749A,0x1FA954DC,0x72276857);
+#endif
+
+// 2017-05-24aRagexeRE
+#if PACKETVER == 20170524
+	packetKeys(0x0B4E03A6,0x2
```

Comment:
You lost the custom key definitions here. Intended?

Reply:
hm, no. i will return it back. thanks